.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: required
 dist: trusty
 
 language: node_js
+language: ruby
```

Comment:
https://stackoverflow.com/questions/27644586/how-to-set-up-travis-ci-with-multiple-languages